lib/PHPExif/Adapter/ImageMagick.php

```diff
@@ -31,6 +32,23 @@ class ImageMagick extends AdapterAbstract
 
     protected string $mapperClass = '\\PHPExif\\Mapper\\ImageMagick';
 
+    /**
+     * Contains the mapping of names to IPTC field numbers
+     */
+    protected array $iptcMapping = array(
+        'iptc:title'       => '2#005',
+        'iptc:keywords'    => '2#025',
+        'iptc:copyright'   => '2#116',
+        'iptc:caption'     => '2#120',
+        'iptc:headline'    => '2#105',
+        'iptc:credit'      => '2#110',
+        'iptc:source'      => '2#115',
+        'iptc:jobtitle'    => '2#085',
+        'iptc:city'        => '2#090',
+        'iptc:sublocation' => '2#092',
+        'iptc:state'       => '2#095',
+        'iptc:country'     => '2#101'
+    );
```

Comment:
I am only curious. Where did you find that values? Did you read the official EXIF specification :astonished: or did you find it somewhere else? Reading the official specs would be impressive. :nerd_face:

Reply:
I'm not that dedicated :-). This is pretty much a copy of IPTC support from Native: https://github.com/LycheeOrg/php-exif/blob/2bd2d3482cca296fc477fe1a07d1f465e145264c/lib/PHPExif/Adapter/Native.php#L70-L83

In general, I would say that EXIF support in Imagick feels very much like a wrapper around the native `exif_read_data`, although I haven't checked if it can do anything more with things like RAWs...